browse/controllers/abs_page.py

```diff
@@ -147,6 +147,9 @@ def get_abs_page(arxiv_id: str) -> Response:
                     response_data["higher_version_withdrawn_submitter"] = _get_submitter(abs_meta.arxiv_identifier,
                                                                                          ver.version)
 
+        response_data["encrypted"] = abs_meta.version_history[abs_meta.version - 1].source_flag.source_encrypted
```

Comment:
Could this be `abs_meta.get_requested_version().source_flag.source_encrypted`?

Reply:
oh theres a function for this! Thats way more convenient :)